WordPress/WordPressTest/UIAlertController+tap.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+
+extension UIAlertController {
```

Comment:
Hi @nheagy! Any reason why you consider this to be dangerous?

This extension is accessible uniquely and only in the Unit Test target. Plus, we can use it in the future on any other test that uses `UIAlertController`.

Reply:
Dangerous because it uses `UnsafeRawPointer` and has knowledge of `UIAlertController`'s implementation. Makes it a touch fragile, for example if and update changes implementation, tests might start crashing instead of failing, and this method may be difficult to trace as the cause. Whereas if its in the same file as the dying tests, it's probably easier to figure out.

I'd say keep it in the file _until_ we want to use it in more tests.

In any case, let's give it a bit of documentation to explain what it's doing?

Reply:
Makes sense.

> I'd say keep it in the file until we want to use it in more tests.

I like that!

I've moved this extension to a simple function and updated the tests.